tests/Api/Response/Parser/FiltersParserTest.php

```diff
@@ -69,7 +78,8 @@ protected function getFiltersParserMock($methods = [])
 
         $filtersParserMock = $this->getMockBuilder(FiltersParser::class)
             ->setConstructorArgs([
-                'libraryCallContract' => $this->libraryCallContract
+                'libraryCallContract' => $this->libraryCallContract,
+                'configRepository' => $this->configRepository
```

Comment:
Just noticed, that we do not test if the configuration is set correctly. Could you add a test for that?